### ISSUE TITLE
Vendored livereload.js cannot be found in 1.5.1

### ISSUE THREAD

**Issue by Fjan** [denisdefreyne/adsf#40]
### Steps to reproduce

1. Using adsf-live version 1.5.1

### Expected behaviour

It should use the vendor livereload.js

### Actual behavior

`Errno::ENOENT: No such file or directory @ rb_file_s_size - ... /adsf-live-1.5.1/lib/adsf/live/../../../vendor/livereload.js (Errno::ENOENT)`

### Details

This probably happens because I did not add the file to the [manifest](https://github.com/denisdefreyne/adsf/blob/main/adsf-live/adsf-live.manifest) in https://github.com/denisdefreyne/adsf/issues/37#issuecomment-2381123455

PR here https://github.com/denisdefreyne/adsf/pull/41

**Comment by Fjan** [comment]
A workaround is to drop livereload.js in your contents folder and patch nanoc to use that version instead:

```ruby
class Rack::LiveReload
  def livereload_source = "/livereload.js"
end
```
(As an aside: This also allows you to provide a more recent version of livereload.js. I simply added the version from the rack-livereload gem from a decade ago. It still works fine, it lacks some improvements, but newer versions tend to require a whole JS toolchain to compile)

**Comment by denisdefreyne** [comment]
Thanks! Will push a fix in a minute.

**Comment by denisdefreyne** [comment]
This is now released in v1.5.2.
